src/Shared/Recruit.Vacancies.Client/Domain/Entities/BannedPhrasesData.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Esfa.Recruit.Vacancies.Client.Domain.Entities
+{
+    public class BannedPhrasesData : ProfanityData
```

Comment:
This doesn't need to inherit from `ProfanityData` does it?

Reply:
Drunken refactoring